apps/src/StudioApp.js

```diff
@@ -130,6 +138,18 @@ var StudioApp = function () {
   this.maxRequiredBlocksToFlag_ = 1;
 
   /**
+  * @type {!TestableBlock[]}
+  */
+  this.recommendedBlocks_ = [];
+
+  /**
+  * The number of recommended blocks to give hints about at any one time.
+  * Set this to Infinity to show all.
+  * @type {!number=}
```

Comment:
hah, I totally missed the `=`; I just copied this from the JSDoc for `maxRequiredBlocksToFlag_`. I'll try to figure out what they're both actually supposed to be